index.js

```diff
@@ -342,7 +344,48 @@ app.post('/api/doorTest', async(req, res) => {
     const {locationid} = req.body;
     await redis.addDoorTestSensorData(req, res);
     await handleSensorRequest(locationid)
-});
+})
+
+app.post('/api/door', async(request, response) => {
+    const coreId = request.body.coreid
+    const location = await db.getLocationIDFromParticleCoreID(coreId)
+    const locationid = location.locationid
```

Comment:
This will cause an error if `locationid` id `null`. We probably just want to log the null and return a decent error code in this case.